src/lib/accessibility/Behaviors/Tab/TabListBehavior.ts

```diff
@@ -0,0 +1,12 @@
+import { Accessibility } from '../../interfaces'
+
+const TabListBehavior: Accessibility = (props: any) => ({
+  attributes: {
+    root: {
+      role: 'tablist',
+      'aria-label': props['aria-label'],
```

Comment:
I think this is not needed. It will come out of the box through the rest/unhandledProps.

Reply:
will remove it, thanks.

Reply:
actually, this behavior (where `rest` or `unhandledProps` are passed down the components' tree) is not guaranteed by consumer of accessibility module. If we would like `accesibility` module being an independent unit, these lines should be here